docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -1,28 +1,37 @@
 [id="minimum-system-requirements_{context}"]
 = Minimum system requirements
 
+[role="_abstract"]
 {prod} has the following minimum hardware and operating system requirements.
 
 [id="minimum-system-requirements-hardware_{context}"]
 == Hardware requirements
 
-{prod} requires the following system resources:
+{prod} is supported only on AMD64 and Intel 64 processor architectures.
+Support for the ARM-based M1 architecture is in development.
```

Comment:
@code-ready/crc-team Please double-check the wording here for me -- I'm not particularly aware of the developments in this area. Should we mention it at all at this point?

Reply:
We have a Q2 goal to deliver M1 support, but this might only be possible for the `{podman}` preset.
Not sure we should make a 'promise' here as the `{ocp}` options depends on availability of `{rhcos}` that can run on Aarch64.

Reply:
I think we should avoid M1 arch as of now since vfkit part is still not in the branch and also we might need some condition for avoid user to run openshift on M1.

Reply:
Okay, I reverted the change to that line. We're back to the original docs statement of "not supported" for the time being.